examples/complex_sys/raze/remote/BUILD.libssh2-sys-0.2.20.bazel

```diff
@@ -43,13 +43,13 @@ cargo_build_script(
     crate_features = [
     ],
     crate_root = "build.rs",
-    # MODIFIED MANUALLY #
-    links = "ssh2",
     data = glob(["**"]) + [
         "@openssl",
     ],
-    # / MODIFIED MANUALLY #
     edition = "2015",
+    # TODO: This must be manually added until a feature in cargo-raze is implemented
```

Comment:
Isn't there a mechanism to have these edits not live directly in the generated files?

Reply:
No, this is the only thing that must continue to be manually added. When I have some free time I can add that functionality to `cargo-raze` though. Seems pretty simple.

Reply:
That's what https://github.com/google/cargo-raze/issues/344 covers

Reply:
@mfarrugi I do feel that this is a good improvement and would reduce the footprint of https://github.com/bazelbuild/rules_rust/pull/500.

It'd be a great help if this could be merged 🙏 I'm happy to address this in the future 😄